service/src/main/resources/db/changesets/20240625_add_workspace_storage_url_to_pipeline_runs.yaml

```diff
@@ -0,0 +1,12 @@
+# Add a workspace_storage_url column to the pipeline_runs table
+databaseChangeLog:
+  - changeSet:
+      id: Add a workspace_storage_url column to the pipeline_runs table
+      author: mma
+      changes:
+        - addColumn:
+            tableName: pipeline_runs
+            columns:
+              - column:
+                  name: workspace_storage_url
```

Comment:
extreeeeeeeeeme nitpick so feel free to ignore, `workspace_storage_container_url`